app/controllers/alert_requests_controller.rb

```diff
@@ -0,0 +1,114 @@
+class AlertRequestsController < ApplicationController
+  layout 'alert_request'
+  before_action :find_partner
+
+  def new
+    @alert_request = AlertRequest.new(
+      partner_id: @partner_id, 
+      tracking_source: @source,
+      tracking_id: @tracking,
+      phone_type: 'mobile',
+    )
+
+    # do we need this?
```

Comment:
Yeah, I think we should add this in. The whole form is really an opt-in but I think we should have this to mimic the other ones in case they want explicit opt-in tracking (and separated partner vs rtv opt-ins)